plugins/main/bridge/bridge.go

```diff
@@ -708,7 +715,9 @@ func cmdAdd(args *skel.CmdArgs) error {
 		}
 
 		if idx == len(retries)-1 {
-			return fmt.Errorf("bridge port in error state: %s", hostVeth.Attrs().OperState)
+			if !n.DisableContainerInterface {
```

Comment:
It does not look to be relevant here.
The "check bridge port state" needs to be bypass if the flag is set to true.

Reply:
Done